debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -428,6 +428,8 @@ tableOption
     | DATA DIRECTORY '='? STRING_LITERAL                                            #tableOptionDataDirectory
     | DELAY_KEY_WRITE '='? boolValue=('0' | '1')                                    #tableOptionDelay
     | ENCRYPTION '='? STRING_LITERAL                                                #tableOptionEncryption
+    | ENCRYPTED '='? encrypted=(YES | NO)                                           #tableOptionEncrypted
```

Comment:
Add the token definition for `YES`, which should fix the failing test.

Reply:
added